apis/python/src/tiledbsc/__init__.py

```diff
@@ -1,6 +1,8 @@
 from .soma_collection import SOMACollection
 from .soma import SOMA
 from .soma_options import SOMAOptions
+from .soma_collection import SOMACollection
+from .soma_slice import SOMASlice
 
 from .tiledb_object import TileDBObject
```

Comment:
should these be exported to the top-level namespace, or should they be internal-only?

Reply:
@bkmartinjr `SOMACollection` is 100% external -- I don't see how it couldn't be -- what am I missing?

`SOMASlice` -- kind of depends on what we suggest/allow people to do with slice results -- which is still a WIP here ...